src/views/layouts/main.hbs

```diff
@@ -3,6 +3,9 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://unpkg.com/tachyons@4.10.0/css/tachyons.min.css"/>
+    <link rel="stylesheet" href="style.css">
+    <img src="./logo.jpg" class="ml5 cover pa2">
```

Comment:
Remove the margin here and put flex here instead!

Reply:
Ok :smiley_cat: